python/ray/tune/experiment.py

```diff
@@ -73,6 +92,12 @@ class Experiment:
             local_dir="~/ray_results",
             checkpoint_freq=10,
             max_failures=2)
+
+    Attributes:
```

Comment:
We consolidated to using "Attributes" instead of "Args": https://github.com/ray-project/ray/pull/22975#discussion_r824311556